app/Resources/public/css/print.css

```diff
@@ -119,3 +93,102 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
+    border-spacing: 0;
+    margin: 0 0 18px;
+}
+
+.data_table td {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".data_table td" to come before selector "table.table td" (no-descending-specificity)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)